### ISSUE TITLE
feat(customization): provide an option to have a submit button for all forms

### ISSUE THREAD

**Issue by SamuelHassine** [LuatixHQ/opencti#6608]
## Use case

In settings, providing an option to have a submit button for all forms

**Comment by nino-filigran** [comment]
I see that: 
- security/organisations is missing a creation/edition form.

Is that the only form missing?

**Comment by SamuelHassine** [comment]
No, I'm talking about everywhere, having a settings parameter (global): "Enable submit buttons", which impacts all the platform and then you have to use the submit button in update forms instead of automatic update when focus out the field.
